src/main/java/core/basesyntax/db/Dao.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.db;
+
+import java.util.Map.Entry;
+import java.util.Set;
+
+public interface Dao {
+    boolean addEntry(String fruitName, int quantity);
+
+    boolean removeEntry(String fruitName);
+
+    boolean isStorageEmpty();
+
+    boolean isFruitPresent(String fruitName);
+
+    int getFruitQuantity(String fruitName);
+
+    Set<Entry<String, Integer>> getAllEntries();
```

Comment:
```suggestion
    boolean addString fruitName, int quantity);

    boolean remove(String fruitName);

    boolean isEmpty();

    boolean contains(String fruitName);

    int getQuantityByName(String fruitName);

    Set<Entry<String, Integer>> getAll();
```
CRUD naming conventions

Reply:
Done as suggested, but we have not studied CRUD and its naming convention yet.